.babelrc

```diff
@@ -35,6 +35,12 @@
       "plugins": [
         "./build/use-lodash-es"
       ]
+    },
+    "rollup": {
+      "plugins": [
+        "./build/use-lodash-es",
+        "external-helpers"
```

Comment:
Well, transform-runtime should only add used helpers by itself. But there can be a tree of helpers using other helpers underneath that. I would just ensure the `{ polyfills: false }` option is used so it doesn't try to polyfill everything. That can be left up to the user.